src/main/java/com/denizenscript/clientizen/objects/EntityTag.java

```diff
@@ -200,6 +206,18 @@ public static void register() {
             }
             return null;
         });
+
+        // <--[tag]
+        // @attribute <EntityTag.is_rendering>
+        // @returns ElementTag
+        // @group Clientizen
```

Comment:
This group doesn't really make sense, everything here is within Clientizen - can probably just remove it? we will probably have some group organization later on, but right now can't really do that with so few tags